ubi/src/forge.rs

```diff
@@ -92,34 +96,41 @@ impl ForgeType {
         Ok(match self {
             ForgeType::GitHub => Box::new(GitHub::new(project_name, tag, api_base_url, token)),
             ForgeType::GitLab => Box::new(GitLab::new(project_name, tag, api_base_url, token)),
+            ForgeType::Forgejo => Box::new(Forgejo::new(project_name, tag, api_base_url, token)),
         })
     }
 
     pub(crate) fn url_base(&self) -> Url {
         match self {
             ForgeType::GitHub => Url::parse(&format!("https://{GITHUB_DOMAIN}")).unwrap(),
             ForgeType::GitLab => Url::parse(&format!("https://{GITLAB_DOMAIN}")).unwrap(),
+            ForgeType::Forgejo => Url::parse(&format!("https://{FORGEJO_DOMAIN}")).unwrap(),
         }
     }
 
     pub(crate) fn api_base_url(&self) -> Url {
         match self {
             ForgeType::GitHub => Url::parse(GITHUB_API_BASE).unwrap(),
             ForgeType::GitLab => Url::parse(GITLAB_API_BASE).unwrap(),
+            // The maintainers of Forgejo is Codeberg, hence why the URL
+            // doesn't align with the name of the Git forge.
+            ForgeType::Forgejo => Url::parse(CODEBERG_API_BASE).unwrap(),
```

Comment:
But the api base changes depending on which forgejo instance you want to talk to?